examples/error-destructure-binding-pattern/error_destructure_binding_pattern.bal

```diff
@@ -17,21 +17,21 @@ public function main() {
     // and assign the values to the relevant variable references.
     // The value of the reason string in the `SampleError` will be assigned to the variable `reason`.
     // The values in the detail mapping will be assigned to the relevant variables.
-    error(reason, info = info, fatal = fatal) = getSampleError();
+    error (reason, info = info, fatal = fatal) = getSampleError();
```

Comment:
I think we don't need the space here. 

If it's a typed binding pattern and we add a space it will be
` error error(reason, info = info, fatal = fatal) = getSampleError();`